msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+        "dependsOn": [
+            "Start Teams App Locally",
+            "Start Add-in Locally"
+        ],
+        "dependsOrder": "sequence"
+    },
+   ```
+
+1. Open the .vscode\launch.json file in the project, which configures the **RUN AND DEBUG** UI in Visual Studio Code and add the following two objects to the top of the "configurations" array.
+
+    ```
+    {
+        "name": "Launch Add-in Outlook Desktop (Edge Chromium)",
+        "type": "msedge",
+        "request": "attach",
+        "port": 9229,
+        "timeout": 600000,
+        "webRoot": "${workspaceRoot}",
+        "preLaunchTask": "Start Add-in Locally",
+        "postDebugTask": "Stop Debug"
+    },
+    {
+        "name": "Launch App and Add-in Outlook Desktop (Edge Chromium)",
+        "type": "msedge",
+        "request": "attach",
+        "port": 9229,
+        "timeout": 600000,
+        "webRoot": "${workspaceRoot}",
+        "preLaunchTask": "Start App and Add-in Locally",
+        "postDebugTask": "Stop Debug"
+    },
+    ```
+
+1. In the "compounds" section of the same file, rename the "Debug in Teams (Edge)" compound to "Launch App Debug (Edge)", and rename the "Debug in Teams (Chrome)" compound to "Launch App Debug (Chrome)".
+
+1. Verify that you can sideload the add-in capability of the Teams app to Outlook with the following steps:
+
+      1. *First, make sure Outlook desktop is closed.*
+      1. In Visual Studio Code, open the Teams Toolkit.
+      1. In the **ACCOUNTS** section, verify that you're signed into Microsoft 365.
+      1. Select **View** | **Run** in Visual Studio Code. In the **RUN AND DEBUG** drop down menu, select the option, **Launch Add-in Outlook Desktop (Edge Chromium)** (*not* any of the "Launch *App* ..." options), and then press F5. The project builds and a Webpack dev-server window opens. This process can take a couple of minutes. Eventually, Outlook desktop opens.
+      1. Open the **Inbox** *of your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
+      1. Select the **Show Taskpane** button and a task pane opens. Select the **Perform an action** button and a small notification appears near the top of the message.
+      1. To stop debugging and uninstall the add-in, select **Run** | **Stop Debugging** in Visual Studio Code. If the Webpack dev-server window doesn't close, open the Visual Studio Code **TERMINAL** in the root of the project and run `npm stop`.
+
+### Run the app and add-in locally at the same time
+
+You can sideload and run the app and the add-in simultaneously, but the debugger can't reliably attach when both are running. So to debug, run only one at a time. 
+
+To debug the app, see the last step of the **Prepare the Teams app project** section.
+
+To debug the add-in, see the last step of the **Edit the tooling configuration files** section.
```

Comment:
Can we please add link to **Edit the tooling configuration files** and update the sentence as suggested below?

To debug the app, refer the last step of **Edit the tooling configuration files**.

Reply:
done